beater/beater.go

```diff
@@ -549,7 +549,8 @@ func (s *serverRunner) run(listener net.Listener) error {
 	}
 	runServer = s.wrapRunServerWithPreprocessors(runServer)
 
-	batchProcessor := make(modelprocessor.Chained, 0, 3)
+	batchProcessor := make(modelprocessor.Chained, 0, 4)
+	batchProcessor = append(batchProcessor, indexDocCountField)
```

Comment:
Should we still be adding this processor when using a non-Elasticsearch output?

Reply:
ah, fair point. I've actually never had to check for that -- how do I best check for that? check to see if `serverRunner.elasticsearchOutputConfig == nil`, and then assume the output is not elasticsearch?